packages/allure-js-commons/src/writers/FileSystemAllureWriter.ts

```diff
@@ -22,6 +22,11 @@ export class FileSystemAllureWriter implements AllureWriter {
     writeFileSync(path, content);
   }
 
+  writeAttachmentPath(targetFileName: string, sourceFilePath: PathLike): void {
```

Comment:
Reading the method name I expect that it writes the path somewhere, but not the actual file.

How about renaming it to `writeAttachmentFromPath`?